src/api/app/models/event/comment.rb

```diff
@@ -32,6 +32,11 @@ def custom_headers
 class Event::CommentForProject < ::Event::Project
   include CommentEvent
   receiver_roles :maintainer
+  after_create :send_to_bus
+
+  def self.message_bus_queue
+    'opensuse.obs.project.comment'
```

Comment:
the prefix needs to be configurable - you might have more than one build service on your bus.

And why only comment events?

Reply:
Nice, so, don't merge it until que change to have a configurable prefix.
And we need to add the queue name for all the events you did that for...

Reply:
> And why only comment events?

Just a proof of concept especially for you! 🌻

Reply:
The prefix is now configurable, see #3934.